src/components/Add_Ride.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState, useEffect } from 'react';
+import Select from 'react-select';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './component.css';
+import AddLocation from './Add_Location';
+import {Button, InputGroup, Label, Input} from 'reactstrap';
+
+
+
+function AddRide() {
+
+  const emptyCabRoute = { location: '', credit: '', sequence_id: '' };
+  var [cabNumberList, setCabNumberList] = useState([]);
+  const [cabNumber, setCabNumber] = useState({});
+  const [timeSlot, setTimeSlot] = useState('');
+  const [cabRoute, setCabRoute] = useState([{...emptyCabRoute}]);
+  
+  cabNumberList = [
+    { cab_id: 1, cab_number: 'MH15MB1234', cab_capacity: 4 },
+    { cab_id: 2, cab_number: 'MH14MN7896', cab_capacity: 3},
+    { cab_id: 3, cab_number: 'MH15MB7456', cab_capacity: 2}
+  ];
```

Comment:
Why this list is not fetch from backend

Reply:
I have Done The changes, Please Review It.